### DISCUSSION TITLE
How to configure a custom VPN profile?

### DISCUSSION THREAD

**Discussion by WatEvsCevs** [haugene/docker-transmission-openvpn#2139] [Q&A] [answered]
I have an Ubuntu VM in the cloud running PiVPN and want to use the .ovpn profile that I downloaded. How do configure my custom VPN profile?

**Comment by pkishino** [comment, marked as answer]
read the documentation
https://haugene.github.io/docker-transmission-openvpn/supported-providers/#using_a_local_single_ovpn_file_from_provider

**Reply by WatEvsCevs** [reply]
Thank you! I found the example ` - /volume1/docker/ipvanish/my-preferred-config-file.ovpn:/etc/openvpn/custom/default.ovpn`
Do I need to remove config, username and password, and what do I input for OpenVPN Provider?

**Reply by pkishino** [reply]
it clearly states what is needed in the docu I linked...
